src/components/homepage/featuredTestimonial.js

```diff
@@ -33,6 +33,7 @@ const ImageSection = styled.div`
   width: 100%;
   overflow: hidden;
   border-radius: 1.25rem;
+  margin-left: 2rem;
```

Comment:
This is one way to do it and it works but instead of adding the padding to the image here, I would increase the gap for the row. So in the container flexbox, I would increase the gap:. 

Also since we're adding space there I would adjust the max-width of this image by 2em.

Reply:
Updated the code to increase the gap in the flexbox container and increase the max-width of the image.